src/OrbitGl/TextBox.h

```diff
@@ -20,6 +20,10 @@ class TextBox {
 
   void SetSize(const Vec2& size) { size_ = size; }
   void SetPos(const Vec2& pos) { pos_ = pos; }
+  void SetPosAndSize(const Vec2& pos, const Vec2& size) {
```

Comment:
Is this necessary for performance reasons? Could this be implemented using `SetSize` and `SetPos`?

Reply:
It was to make the code shorter, but I don't need it anymore actually so I removed it in favor of using the individual SetPos and SetSize.